tests/CMakeLists.txt

```diff
@@ -76,7 +76,7 @@ if(NOT (MSVC AND MSVC_VERSION LESS 1920))
   add_compile_test(readme_examples_noexceptions readme_examples_noexceptions.cpp quicktests)
 
   # Compile tests that *should fail*
-  add_compile_test(readme_examples_will_fail_with_exceptions_off readme_examples.cpp FALSE quicktests)
+  add_compile_test(readme_examples_will_fail_with_exceptions_off readme_examples.cpp FALSE quicktests) # <--- those two arguments at the end, are leftovers, right?
```

Comment:
Yes, they are, good catch! You may remove them if you so desire :) Otherwise, I have a patch upcoming that will remove them.

Reply:
removed